src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25038.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25038 : _IssuesUITest
+	{
+		public Issue25038(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "MAUI Entry in Windows always shows ClearButton if initially hidden and shown even if ClearButtonVisibility set to 'Never'";
+
+		[Test]
+		[Category(UITestCategories.Entry)]
+		[FailsOnMac]
+		public void VerifyEntryClearButtonVisibility()
+		{
+			App.WaitForElement("button");
+			App.Tap("button");
+			VerifyScreenshot();
```

Comment:
Triggered the build, it should generate the reference snapshot for this test.

Reply:
Ok, already added the test snapshot.

Reply:
Thanks for the images